static/data/2022-mayors-estimate-fullgeneralfund.json

```diff
@@ -0,0 +1,223 @@
+{
+	"fund_structure": [{
+			"name": "Environment and Sustainability",
+			"children": [{
+					"name": "Office of Sustainability",
+					"total": 1291567
+				},
+				{
+					"name": "Department of Public Health - Environment",
+					"total": 2354576
+				},
+				{
+					"name": "Department of Public Health - Air Quality",
+					"total": 1111059
+				},
+				{
+					"name": "Division of Waste Disposal",
+					"total": 35248055
+				}
+			]
+		},
+		{
+			"name": "Emergency Services",
+			"children": [{
+					"name": "Fire",
+					"total": 100227328
+				},
+				{
+					"name": "Emergency Medical Service",
+					"total": 34781134
+				},
+				{
+					"name": "Animal Care and Control",
+					"total": 3720253
+				},
+				{
+					"name": "Department of Public Safety - Director's Office",
+					"total": 8011064
+				}
+			]
+		},
+		{
+			"name": "Community Services",
+			"children": [{
+					"name": "Recreation",
+					"total": 15539471
+				},
+				{
+					"name": "Parks Maintenance & Properties",
+					"total": 18077742
+				},
+				{
+					"name": "Landmarks Commission",
+					"total": 274190
+				},
+				{
+					"name": "Department of Aging",
+					"total": 1826940
+				},
+				{
+					"name": "Office of Intervention, Prevention, and Opportunity",
+					"total": 4657799
+				},
+				{
+					"name": "Office of Equal Opportunity",
+					"total": 1109519
+				},
+				{
+					"name": "Community Relations Board",
+					"total": 2313582
+				}
+			]
+		},
+		{
+			"name": "Health",
+			"children": [{
+					"name": "Department of Public Health - Director's Office",
+					"total": 2522042
+				},
+				{
+					"name": "Department of Public Health - Health",
+					"total": 6802300
+				},
+				{
+					"name": "Health, Equity & Soc. Justice",
+					"total": 544024
+				}
+			]
+		},
+		{
+			"name": "Infrastructure and Housing",
+			"children": [{
+					"name": "Office of Capital Projects",
+					"total": 7386567
+				},
+				{
+					"name": "Public Works Administration",
+					"total": 3820908
+				},
+				{
+					"name": "Non-Departmental: Subsidy to Street Construction",
+					"total": 8464575
+
+				},
+				{
+					"name": "Community Development Director's Office",
+					"total": 2366508
+
+				},
+				{
+					"name": "Building Standards and Appeals",
+					"total": 169435
+				},
+				{
+					"name": "City Planning Commission",
+					"total": 2605289
+				},
+				{
+					"name": "Zoning Appeals",
+					"total": 253894
+				},
+				{
+					"name": "Department of Building & Housing",
+					"total": 14348075
+				},
+				{
+					"name": "Department of Public Works - Property Management",
+					"total": 8895036
+				},
+				{
+					"name": "Department of Public Works - Parking Facilities",
+					"total": 1054821
+				},
+				{
+					"name": "Department of Public Works - Division of Traffic Engineering",
+					"total": 3989151
+				}
+			]
```

Comment:
I reviewed values for each of the line items (e.g. 'parking facilities') whether they match up with what's in the 2022 budget book (pgs. 51-53 of the PDF); they all match; this is as far as I've gotten. As noted in that [google doc ](https://docs.google.com/document/d/1SRyMOYIv_RTlDsTDqVpPwxjUElxs1fJagAgLUjcmcHc/edit); "Health, Equity & Soc. Justice" is not explicitly mentioned in the expense summary (pg. 51-53) so I wondered maybe that was the cause for the discrepancy ; but I don't think that alone is it; even if omit that from our; there's still a difference between 653,647,414 - TOTAL EXECUTIVE BRANCH (pg. 53) and what's on the frontpage of our website (653,275,220 as of this commit.